profile/profile.go

```diff
@@ -31,13 +31,14 @@ import (
 
 // Profile is an in-memory representation of profile.proto.
 type Profile struct {
-	SampleType        []*ValueType
-	DefaultSampleType string
-	Sample            []*Sample
-	Mapping           []*Mapping
-	Location          []*Location
-	Function          []*Function
-	Comments          []string
+	SampleType            []*ValueType
+	DefaultSampleType     string
+	Sample                []*Sample
+	Mapping               []*Mapping
+	Location              []*Location
+	Function              []*Function
+	Comments              []string
+	InferredNumLabelUnits map[string]string
```

Comment:
It is a bit strange that we attach this to the profile only using it in the reporting flow, not in the parsing. Can we pass the information about the inferred units separately from the profile?